src/main/webapp/WEB-INF/jsp/links/configLinks.jsp

```diff
@@ -0,0 +1,325 @@
+    <li class="list-group-item list-group-item-action" draggable="true">
+        <i class="fa fa-pull-left fa-fw fa-3x" aria-hidden="true"></i>
+        <a class='fa-pull-right close-btn'><i class="fa fa-pull-right fa-close text-danger" aria-hidden="true"></i></a>
+        <dl>
+            <dt class="title"></dt>
+            <dd class="description"></dd>
+            <dd class="url"></dd>
+            <dd class="groups"></dd>
+        </dl>
+    </li>
+</template>
+
+<script type="text/javascript">
+    var ${n} = {}; // create a unique variable for our JS namespace
+
+    (function($) {
+        var default_link = {title: "Important Site", description: "A few words about this link", icon: "fa-link",
+                            url: "https://www.yourapp.edu/", groups: []};
+        var drag_source;
+
+        var cloneDefaultLink = function() {
+            var newlink = {};
+            for (var i in default_link)
+                newlink[i] = default_link[i];
+            return newlink;
+        }
+
+        var createLinkListItem = function(link) {
+            var template = document.getElementById('${n}_link_item');
+            var item = document.importNode(template.content, true);
+            item.querySelector('i').classList.add(link.icon);
+            var dl = item.querySelector('dl');
+            dl.querySelector('dt.title').textContent = link.title;
+            dl.querySelector('dd.description').textContent = link.description;
+            dl.querySelector('dd.url').textContent = link.url;
+            dl.querySelector('dd.groups').textContent = link.groups;
+            return item;
+        }
+
+        var setupLi = function(li, link) {
+            // post appendChild work
+            li.link = link;
+            li.addEventListener('click', selectItem);
+            li.addEventListener('dragstart', dragstart);
+            li.addEventListener('dragover', dragover);
+            li.addEventListener('drop', dropped);
+            li.querySelector('.close-btn').addEventListener('click', removeLink);
+        }
+
+        var addItem = function(e) {
+            var app = document.getElementById('${n}config');
+            var link = cloneDefaultLink();
+            app.querySelector('ul').appendChild(createLinkListItem(link));
+            var last;
+            _.forEach(document.getElementById('${n}config').querySelectorAll("li"), function(li) {
+                li.classList.remove('active');
+                last = li;
+            });
+            last.classList.add('active');
+            setupLi(last, link);
+            populateEditForm(last);
+        }
+
+        var parentLi = function(target) {
+            var li = target;
+            while (!li.matches('li')) {
+                li = li.parentNode;
+            }
+            return li;
+        }
+
+        var selectItem = function(e) {
+            _.forEach(document.getElementById('${n}config').querySelectorAll("li"), function(li) {
```

Comment:
:art: replace `"` with `'` to remain consistent with quotes in the rest of the file